src/interfaces/IGovLst.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Why are we creating an interface file here, then importing it into the script, instead of just using the actual `GovLst` file we have in this repo?

Reply:
I was under the wrong impression that it would save deployment gas, but that's not the case since we're not inheriting. Removed: https://github.com/withtally/stGOV/pull/91/commits/cc0ade97100ed514eaec50f65c2212b794355c33